cmd/k8s-operator/operator_test.go

```diff
@@ -54,7 +60,7 @@ func TestLoadBalancerClass(t *testing.T) {
 		},
 	})
 
-	expectRequeue(t, sr, "default", "test")
+	expectReconciled(t, sr, "default", "test")
```

Comment:
Same as above, was is this related to the logging or just a missed change from the previous PR?

Reply:
Yeah just a missed reconcile change.

Reply:
👍🏾